go/worker/client/worker.go

```diff
@@ -185,13 +196,14 @@ func New(
 	}
 
 	w := &Worker{
-		enabled:      enabled,
-		commonWorker: commonWorker,
-		registration: registration,
-		runtimes:     make(map[common.Namespace]*committee.Node),
-		quitCh:       make(chan struct{}),
-		initCh:       make(chan struct{}),
-		logger:       logging.GetLogger("worker/client"),
+		enabled:             enabled,
+		commonWorker:        commonWorker,
+		registration:        regWorker,
```

Comment:
```suggestion
		registrationWorker:        registrationWorker,
```
Not the best name, but just to be consistent with other code.